apps/style/applab/style.scss

```diff
@@ -1,5 +1,4 @@
 @import "color";
-@import "colpick.css";
```

Comment:
This looks like the same thing as the surrounding lines, but while the rest of these cause a SASS inline include, this is left as a raw CSS `@include` directive in the final source, which we should avoid if we intend to use fingerprinted assets.  Since this is an applab-only asset, I've moved it into _blockly.html.haml with the other applab style dependencies.